selfdrive/sensord/rawgps/rawgpsd.py

```diff
@@ -112,7 +120,11 @@ def setup_quectel(diag: ModemDiag):
   mmcli("--command='AT+QGPSCFG=\"dpoenable\",0'")
   # don't automatically turn on GNSS on powerup
   mmcli("--command='AT+QGPSCFG=\"autogps\",0'")
-  mmcli("--location-enable-gps-raw --location-enable-gps-nmea")
+  mmcli("--command='AT+QGPSSUPLURL=\"supl.google.com:7275\"'")
+
+  mmcli("--command='AT+QGPSCFG=\"outport\",\"usbnmea\"'")
+  if not gps_enabled():
+    mmcli("--command='AT+QGPS=1'")
```

Comment:
instead of this, we should check if enabled as the first step of the setup and turn off at that point. I remember something about not taking any configuration until GPS is off in the docs